easybuild/easyconfigs/r/R/R-3.6.3-foss-2020a.eb

```
@@ -0,0 +1,2309 @@
+    }),
+    ('clisymbols', '1.2.0', {
+        'checksums': ['0649f2ce39541820daee3ed408d765eddf83db5db639b493561f4e5fbf88efe0'],
+    }),
+    ('ini', '0.3.1', {
+        'checksums': ['7b191a54019c8c52d6c2211c14878c95564154ec4865f57007953742868cd813'],
+    }),
+    ('gh', '1.1.0', {
+        'checksums': ['de9faf383c3fe5e87a75391d82cf71b1331b3c80cd00c4203146a303825d89ad'],
+    }),
+    ('usethis', '1.5.1', {
+        'checksums': ['9e3920a04b0df82adf59eef2c1b2b4d835c4a757a51b3c163b8fc619172f561d'],
+    }),
+    ('DT', '0.12', {
+        'checksums': ['239906a884a485fd09e50bf10a52268f62ffd4ba9569499c4d92055511120901'],
+    }),
+    ('rex', '1.1.2', {
+        'checksums': ['bd3c74ceaf335336f5dd04314d0a791f6311e421a2158f321f5aab275f539a2a'],
+    }),
+    ('covr', '3.5.0', {
+        'checksums': ['cb919912018130164a40803ac573a37dde2186678c058c03c6303d79604979df'],
+    }),
+    ('devtools', '2.2.2', {
+        'checksums': ['e988f435fbb5cfc89c4a4755a999dd08f176dd63151dc340e3bf5d1a91140bae'],
+    }),
+    ('Rook', '1.1-1', {
+        'checksums': ['00f4ecfa4c5c57018acbb749080c07154549a6ecaa8d4130dd9de79427504903'],
+    }),
+    ('Cairo', '1.5-11', {
+        'checksums': ['af23b16b05deae20d7b28e04cee1c256034e1c216337ff44378fd8f258bf33cd'],
+    }),
+    ('RMTstat', '0.3', {
+        'checksums': ['81eb4c5434d04cb66c749a434c33ceb1c07d92ba79765d4e9233c13a092ec2da'],
+    }),
+    ('Lmoments', '1.3-1', {
+        'checksums': ['7c9d489a08f93fa5877e2f233ab9732e0d1b2761596b3f6ac91f2295e41a865d'],
+    }),
+    ('distillery', '1.0-6', {
+        'checksums': ['4910e2952f767c1062d7cbe648c90a97009e2b3da316c6b33f6d022cd38b23d6'],
+    }),
+    ('extRemes', '2.0-11', {
+        'checksums': ['75fbdeef677c81cf5661b8df3df4090c55f53e9bb96bb138b498eb0fbbf5af42'],
+    }),
+    ('pixmap', '0.4-11', {
+        'checksums': ['6fa010749a59cdf56aad9f81271473b7d55697036203f2cd5d81372bcded7412'],
+    }),
+    ('tkrplot', '0.0-24', {
+        'checksums': ['2873630a37d7ae1e09a5803d9a89ca0494edd83526c7b1860d9246543722f311'],
+    }),
+    ('misc3d', '0.8-4', {
+        'checksums': ['75de3d2237f67f9e58a36e80a6bbf7e796d43eb46789f2dd1311270007bf5f62'],
+    }),
+    ('multicool', '0.1-11', {
+        'checksums': ['1c907e64af2ac39facdf431a5691e69649f64af1f50e198ae39da5bf30026476'],
+    }),
+    ('plot3D', '1.3', {
+        'checksums': ['b9e4ec2789e34ad249318900e186868650e1a33466b385cb492a45466db3dfc9'],
+    }),
+    ('plot3Drgl', '1.0.1', {
+        'checksums': ['466d428d25c066c9c96d892f24da930513d42b1bdf76d3b53628c3ba13c3e48a'],
+    }),
+    ('OceanView', '1.0.5', {
+        'checksums': ['c16e1bed97f4ede46dc017fdd6bd7575d925b57bd2601317bd3ad2357609f885'],
+    }),
+    ('ks', '1.11.7', {
+        'checksums': ['6a6d9c2366e85a4c6af39b798f3798d20a42615ddfcebcedf6cf56087cdfd2b8'],
+    }),
+    ('logcondens', '2.1.5', {
+        'checksums': ['72e61abc1f3eb28830266fbe5b0da0999eb5520586000a3024e7c26be93c02eb'],
+    }),
+    (
```

@lexming could you please add the mRMRe package to the list?
```suggestion
    }),
    ('mRMRe', '2.1.0', {
        'checksums': ['fe23c5c1e7b5b653e0358e98f25ebd8c0c74c6c871606d1b24cd02a5534181d4'],
    }),
```

---

@smoors sure :+1: